internal/support/scheduler/router.go

```diff
@@ -0,0 +1,231 @@
+//
+// Copyright (c) 2018 Tencent
+//
+// Copyright (c) 2017 Dell Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package scheduler
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/edgexfoundry/edgex-go/pkg/models"
+	mux "github.com/gorilla/mux"
+	"io"
+	"io/ioutil"
+	"net/http"
+)
+
+func LoadRestRoutes() http.Handler {
+	r := mux.NewRouter()
+	b := r.PathPrefix("/api/v1").Subrouter()
+
+	//ping
+	b.HandleFunc("/ping", ping).Methods(http.MethodGet)
+
+	// meta
+	b.HandleFunc("/info/{name}", info).Methods(http.MethodGet)
+
+	// callback
+	b.HandleFunc("/callbacks", addCallbackAlert).Methods(http.MethodPost)
+	b.HandleFunc("/callbacks", updateCallbackAlert).Methods(http.MethodPut)
+	b.HandleFunc("/callbacks", removeCallbackAlert).Methods(http.MethodDelete)
+
+	return r
+}
+
+const (
+	ContentTypeKey       = "Content-Type"
+	ContentTypeJsonValue = "application/json; charset=utf-8"
+	ContentLengthKey     = "Content-Length"
+)
```

Comment:
These same strings are used pretty widely as literals. Consider moving these constants to internal/constants.go

Reply:
moved local const to scheduler const.go